src/components/config-tab.js

```diff
@@ -1,34 +1,38 @@
 'use strict';
 
-const _ = require('lodash');
 const React = require('react');
+const { createContainer } = require('sovereign');
 
+const configStore = require('../stores/configStore');
+const {
+  fetchConfig,
+  setConfig
+} = require('../actions/configActions');
 const Checkbox = require('../primed/checkbox');
 
 class ConfigTab extends React.Component {
   constructor(...args){
     super(...args);
-    this.state = { autoFill: false };
-    this.toggle = this.toggle.bind(this);
   }
   componentDidMount(){
-    chrome.storage.sync.get('autoFill', (res)=>{
-      this.setState({ autoFill: res.autoFill });
-    });
+    fetchConfig('autoFill');
   }
-  toggle(){
-    this.setState({ autoFill: event.target.checked });
-    chrome.storage.sync.set({ autoFill: event.target.checked });
-    if(event.target.checked){
-      chrome.tabs.query({ url: 'https://github.com/*/*' }, function(tabs){
-        _.forEach(tabs, function(tab){
-          chrome.tabs.sendMessage(tab.id, { fillPR: true });
-        });
-      });
+  config(){
+    setConfig(event.target.checked);
```

Comment:
event isn't defined in the parameters.  did this pass linting?  need to figure out why if it did